.github/workflows/pkgdown-dev2main.yaml

```diff
@@ -33,88 +33,40 @@ jobs:
       - name: Generate sample reports
         shell: Rscript {0}
         run: |
-          devtools::load_all()
+          devtools::load_all(dependencies = T)
```

Comment:
`pkgload::load_all()` (called via `devtools::load_all()` doesn't have a `dependencies` argument. This should throw an error. Were you thinking of `devtools::install()`?

Reply:
oh whoops- that can actually just go away- i was thinking about addressing #1804, but then stopped part-way through and forgot to get rid of all the evidence. but yeah- i was probably thinking of `install` here.